libp2p/protocols/secure/noise.nim

```diff
@@ -464,7 +464,7 @@ method handshake*(p: Noise, conn: Connection, initiator: bool): Future[SecureCon
       var
         failedKey: PublicKey
       discard extractPublicKey(conn.peerInfo.peerId, failedKey)
-      debug "Noise handshake, peer infos don't match!", initiator, dealt_peer = $conn.peerInfo.id, dealt_key = $failedKey, received_peer = $pid, received_key = $remotePubKey
+      trace "Noise handshake, peer infos don't match!", initiator, dealt_peer = $conn.peerInfo.id, dealt_key = $failedKey, received_peer = $pid, received_key = $remotePubKey
```

Comment:
generally it's not a bad idea to keep weird situations like this on `debug` - ie they shouldn't happen really if peers are honest and working so it shouldn't create volume

Reply:
Ok, I'm not opposed to it, the only issue for me personally is when running/debugging tests, it clutters the output too much.